replication/row_event.go

```diff
@@ -1018,6 +1018,13 @@ func (e *RowsEvent) decodeExtraData(data []byte) (err2 error) {
 }
 
 func (e *RowsEvent) DecodeData(pos int, data []byte) (err2 error) {
+	if e.compressed {
+		data, err2 = DecompressMariadbData(data[pos:])
+		if err2 != nil {
+			return
```

Comment:
sorry we have enabled aggressive linter rules, you should be able to add `//nolint:nakedret` in above line to ignore it.